shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.mm

```diff
@@ -307,6 +310,15 @@ - (BOOL)isLiveTextInputAvailable {
   return [[self textField] canPerformAction:@selector(captureTextFromCamera:) withSender:nil];
 }
 
+- (void)showLookUpViewController:(NSString*)term {
+  UIViewController* engineViewController = [_engine.get() viewController];
+  UIReferenceLibraryViewController* referenceLibraryViewController =
+      [[[UIReferenceLibraryViewController alloc] initWithTerm:term] autorelease];
+  [engineViewController presentViewController:referenceLibraryViewController
```

Comment:
Note based on offline discussion:

If we try to present a view controller while another is presenting, iOS will prevent it and log a warning like:
```
[Presentation] Attempt to present <UIReferenceLibraryViewController: 0x12c968a70> on <FlutterViewController: 0x12d817e00> (from <FlutterViewController: 0x12d817e00>) which is already presenting <UIReferenceLibraryViewController: 0x12ca57180>.
```